string filename
float64 rotation # in degrees
